src/components/Savings/SavingsCard/SavingsCardTableMobile/index.tsx

```diff
@@ -14,19 +14,29 @@ import sendGa from 'functions/sendGa'
 
 export const SavingsCardTableMobile = ({
     account,
+    requiredChain,
     hasBalance,
     headings,
     toggleModal,
 }: {
     account: string
+    requiredChain: number
     hasBalance: boolean | undefined
     headings: HeadingCopy
     toggleModal: (type?: ModalType) => void
 }): JSX.Element => {
     const { i18n } = useLingui()
-    const { stats, error } = useStakerInfo(10, account)
+    const { stats, error } = useStakerInfo(42220, 10, account)
     const getData = sendGa
 
+    const toggleSavingsModal = useCallback(
```

Comment:
see comments above. modal button component could be reused here too

Reply:
Replaced all action buttons around savings